### DISCUSSION TITLE
dlt-system process with signal SIGABRT, Aborted.

### DISCUSSION THREAD

**Discussion by amosxc** [COVESA/dlt-daemon#672] [Q&A]
(gdb)
#0  0x0000ffffbec5c444 in poll () from /lib/libc.so.6
#1  0x0000000000404d50 in poll (__timeout=-1, __nfds=5, __fds=0xffffd2889b48) at /usr/include/bits/poll2.h:39
#2  start_dlt_system_processes (config=config@entry=0xffffd2889bf8) at /usr/src/debug/dlt-daemon/2.18.8+2.18.9gitAUTOINC+6a3bd901d8-r0/git/src/system/dlt-system-process-handling.c:276
#3  0x0000000000403970 in main (argc=<optimized out>, argv=<optimized out>) at /usr/src/debug/dlt-daemon/2.18.8+2.18.9gitAUTOINC+6a3bd901d8-r0/git/src/system/dlt-system.c:118

i not find abort() in above backtrace, please have a look why have recieved SIGABRT.

**Comment by minminlittleshrimp** [comment]
Hello @amosxc 
Not much input for maintainer to solve.
Any corefile can be provided from you?
Any reproduced steps can be listed here?
Have you observed this issue with v2.18.10?
